searches/binary_search.py

```diff
@@ -298,8 +298,8 @@ def binary_search_by_recursion(
 
 
 if __name__ == "__main__":
-    user_input = input("Enter numbers separated by comma:\n").strip()
-    collection = sorted(int(item) for item in user_input.split(","))
+    user_input = input("Enter numbers separated by space\n")
+    collection = list(map(int, user_input.split()))
```

Comment:
Your edit changes the behavior of this line. Before, `collection` would be always sorted, but your change makes it so that it's no longer sorted.

Also, this should use a list comprehension rather than `map`.